bin/lib/run_from_docker

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env bash
+
+# DOC: Run scripts from Docker.
+
+# Usage:
+# bin/build-cloud-deployment
+# cloud/shared/bin/run-cloud-deployment <command> <tag> <config_file>
+
+export COMMAND="${1:-deploy}"
+export IMAGE_TAG="${2:-latest}"
+export CONFIG="${3:-civiform_config.sh}"
```

Comment:
Ah, I see now where CONFIG is set. Though still don't fully understand why we need 2 variables.